js/src/components/contentAnalysis/ReadabilityAnalysis.js

```diff
@@ -48,9 +48,13 @@ ReadabilityAnalysis.propTypes = {
  */
 function mapStateToProps( state, ownProps ) {
 	const marksButtonStatus = ownProps.hideMarksButtons ? "disabled" : state.marksButtonStatus;
+	let results = null;
+	if ( typeof( state.analysis.readability ) !== "undefined" ) {
```

Comment:
I'm not sure if that is better, but we use Lodash's isUndefined for these a lot.